### PR TITLE
Fix typos in  `psconvert` in the file `figure.py`

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR aims to fix typos in the API documentation of [`pygmt.Figure.psconvert`](https://www.pygmt.org/dev/api/generated/pygmt.Figure.psconvert.html#pygmt.Figure.psconvert) in the file `figure.py`.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by yvonnefroehlich** [comment]
This PR is more or less complete from my side. 

For the sake of consistency I want[ed] to add dots at the end of the bullet points in the section ['main features'](https://www.pygmt.org/dev/api/index.html#main-features) of the API References. However, I could not figure out where / in which file I have to make the change. In the `pygmt/doc/index.rst` file, I only found:

``` 
.. _api:
API Reference
=============
.. automodule:: pygmt
.. currentmodule:: pygmt
Plotting
```
I would be happy if one could help me out :slightly_smiling_face: . If people agree this change makes sense, I can open a new PR or add this to this PR.

File: pygmt/figure.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def psconvert(self, icc_gray=False, **kwargs):
         PDF, PNG, PPM, SVG, TIFF) using GhostScript.
 
         If no input files are given, will convert the current active figure
-        (see :func:`pygmt.figure`). In this case, an output name must be given
+        (see :func:`pygmt.Figure`). In this case, an output name must be given
```
Comment (seisman) [inline pygmt/figure.py:148]: Strictly speaking, `pygmt.Figure` is a class, not a function.
```suggestion
        (see :class:`pygmt.Figure`). In this case, an output name must be given
```

Comment (yvonnefroehlich) [reply pygmt/figure.py:148]: Thanks @seisman! I made this change alread in PR #2043 https://github.com/GenericMappingTools/pygmt/pull/2043/commits/ef169ec9e33c5b37d35e790783526f336734ed54.

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
> This PR is more or less complete from my side. 
> 
> For the sake of consistency I want[ed] to add dots at the end of the bullet points in the section ['main features'](https://www.pygmt.org/dev/api/index.html#main-features) of the API References. However, I could not figure out where / in which file I have to make the change. In the `pygmt/doc/index.rst` file, I only found:
> 
> ```
> .. _api:
> API Reference
> =============
> .. automodule:: pygmt
> .. currentmodule:: pygmt
> Plotting
> ```
> 
> I would be happy if one could help me out 🙂 . If people agree this change makes sense, I can open a new PR or add this to this PR.

They are here https://github.com/GenericMappingTools/pygmt/blob/main/pygmt/__init__.py.

**Review by michaelgrund — APPROVED** [review]

**Comment by yvonnefroehlich** [comment]
> They are here https://github.com/GenericMappingTools/pygmt/blob/main/pygmt/__init__.py.

Thanks @seisman! I have made the change in PR #2055.

**Pull request merged**